### ISSUE TITLE
SVGO 3.3.0 stopped working in the AWS environment with Node.js 20

### ISSUE THREAD

**Issue by turakvlad** [svg/svgo#1990]
We used SVGO 3.2.0 in the AWS environment (Lambda functions) with Node.js 20 as a runtime, which worked well. When we upgraded to 3.3.0, it stopped working:

<img width="1167" alt="CleanShot 2024-05-08 at 14 29 49@2x" src="https://github.com/svg/svgo/assets/7996184/7c888edf-e4e0-483d-a0a6-8dd9589ee351">

**Comment by XhmikosR** [comment]
See #1991

**Comment by SethFalco** [comment]
Hey hey!

Sorry about that, when creating the CJS bundle, I only added a test for the `optimize` function, and didn't add explicit tests for the other exports. This led to me missing the fact that some exports were missing from the CJS bundle.

This has been resolved by a contributor and releases in [v3.3.1](https://github.com/svg/svgo/releases/tag/v3.3.1).

I apologize for the trouble! If you have any other issues, feel free to open another issue.
